general/community/meetings/index.md

```diff
@@ -13,15 +13,15 @@ Developer meetings are open to anyone interested in Moodle development.
 
 :::important
 
-Our next developer meeting will be on Tuesday 18 October 2022 at 08:00 UTC.
-Details: [Developer meeting October 2022](./202210.md)
+Our next developer meeting will be in December (date tbc).
 
 :::
 
 ## Past meeting notes
 
 ### 2022
 
+- [Developer meeting October 2022](./202210.md)
```

Comment:
This change was also included in Andrew's issue yesterday (so you should remove it from here, because, as you'll see, there are some conflicts with it).